model/src/main/java/org/gluu/oxtrust/api/organization/ManagerGroup.java

```diff
@@ -0,0 +1,28 @@
+package org.gluu.oxtrust.api.organization;
+
+public class ManagerGroup {
```

Comment:
Are we really needed separate bean for Manager Group? Why we can't add group type enum to generic group bean?